src/main/kotlin/com/team4099/robot2022/subsystems/Wheel.kt

```diff
@@ -107,24 +109,23 @@ class Wheel(
       // ${directionSensor.getRawPosition()}")
```

Comment:
ok so this whole method is what I was saying to change to run the control loop on the rio. What we were doing previously was running the control loop on the Spark MAX, which let us run that loop at a faster update rate by using the NEO 550 internal encoder. But this likely also contributed to the jittering effect we saw, because it was possible for that motor to spin a decent amount  without turning the actual module (this is called backlash in the gearbox). So the issue is if you use the internal encoder of the motor, you can see jitter more easily because there is a small amount the motor can turn without turning the module itself.

instead, if we run the control loop on the rio, we can use the absolute encoder directly. So if we create a `ProfiledPIDController` in the `Wheel` class, we can use that controller to generate the voltages we need to apply to the motor, and it can calculate that using the values we read from our `AngularMechanismSensor` above. We can then also add in feedforward which we can calculate by characterizing the steering motor (for now all you need is to create the constants and add them in similarly to how we do for the drive)

Reply:
actually thinking about it further, because analog potentiometers can't read velocity, we may encounter some issues with characterizing the steering. so for now leave this part as is, though all of the other analog potentiometer related changes still need to happen